src/main/java/com/internet/shop/model/User.java

```diff
@@ -10,12 +10,16 @@
     private byte[] salt;
     private Set<Role> roles;
 
-    public User(String name, String login, String password) {
-        this.name = name;
+    public User(String login, String password) {
         this.login = login;
         this.password = password;
     }
 
+    public User(String name, String login, String password) {
+        this(login, password);
+        this.name = name;
+    }
+
     public User(Long userId, String name, String login, String password, Set<Role> roles) {
         this.userId = userId;
```

Comment:
and here?

Reply:
done

Reply:
you have constructor with three of these parameters(name, login, password)